css/decanter.css

```diff
@@ -991,7 +991,7 @@ table {
   border-right: 0; }
 
 html {
-  font-family: "Source Sans Pro", "Helvetica Neue", "Helvetica", "Roboto", "Arial", sans-serif;
+  font-family: "Source Sans Pro", "Helvetica Neue", "Helvetica", "Arial", sans-serif;
```

Comment:
Looks like Roboto went away here. Was this intensional?

Reply:
The fonts after `"Source San Pro"` here are generated by the `$font-stack-helvetica`font stack variable. These are fallbacks. Bourbon has updated these.

Reply:
Thanks for confirming.